torchgeo/samplers/batch.py

```diff
@@ -111,8 +114,15 @@ def __init__(
                 bounds.maxx - bounds.minx >= self.size[1]
                 and bounds.maxy - bounds.miny >= self.size[0]
             ):
+                if bounds.area > 0:
+                    rows, cols = tile_to_chips(bounds, self.size)
+                    self.length += rows * cols
+                else:
+                    self.length += 1
```

Comment:
also, why is length incremented for a hit that has area <= 0?

Reply:
This is for point datasets like GBIF/iNaturalist/EDDMapS